mmdeploy/codebase/mmocr/models/text_detection/panet_trt_fp16.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Does not need to add `trt_fp16` in the file name even when your are solving that problem.

Reply:
maybe we should move this part to document as one of reference solution since this rewriting may only work on a specific checkpoint.